healvis/observatory.py

```diff
@@ -230,14 +283,13 @@ def vis_calc(self, pcents, tinds, shell, vis_array, Nfin, beam_pol='pI'):
             with Nfin.get_lock():
                 Nfin.value += 1
             if mp.current_process().name == 0:
-                #        print('Mem: {}GB'.format(memory_usage_GB))
-                #        sys.stdout.flush()
                 if Nfin.value > 0:
+                    dt = (time.time() - self.time0)
                     sys.stdout.write('Finished: {:d}, Elapsed {:.2f}min, Remain {:.3f}hour, MaxRSS {}GB\n'.format(
                         Nfin.value, dt / 60., (1 / 3600.) * (dt / float(Nfin.value)) * (self.Ntimes - Nfin.value), memory_usage_GB))
                     sys.stdout.flush()
 
-    def make_visibilities(self, shell, Nprocs=1, beam_pol='pI'):
+    def make_visibilities(self, shell, Nprocs=1, times_jd=None, beam_pol='pI'):
```

Comment:
if `vis_calc` is now a hidden function, this probably needs a more fleshed-out doc-string with input parameter types etc.